### DISCUSSION TITLE
Show org structure, role and team when clicking on someone´s name

### DISCUSSION THREAD

**Discussion by teresa-novoa-zz** [mattermost/community#146] [Practical Ideas]
Show org structure, role, and team when clicking on someone´s name

**Comment by esethna** [comment]
We have a "position" field but it is a very manual process or linked to LDAP: 
![image](https://user-images.githubusercontent.com/14333569/123487609-16099d80-d5c3-11eb-9576-5fe90cc05362.png)

Agree this could certainly be more interactive and help. I find myself opening Bamboo often

**Comment by jespino** [comment]
I think we can solve that with saml too, but only would be updated on login (I think)

**Reply by chetanyakan** [reply]
IIRC, the SAML/LDAP system console fields allow you to disable the manual editing of users position.

**Reply by chetanyakan** [reply]
Now that I read it again, you mean the case when  someone's position changes on the login provider.
That might require re-login, but I wonder is there something like LDAP-sync for SAML?
